src/model/page-config-resolver.js

```diff
@@ -291,6 +358,64 @@ class MicrodataParser {
     this.win_ = win;
   }
 
+  discoverAccess_(prop) {
+    if (prop.getAttribute('itemprop') == 'isAccessibleForFree') {
+      const value = prop.content || prop.textContent;
+      if (typeof value == 'boolean') {
```

Comment:
Can it ever really be a `boolean`?

Reply:
JSON-LD is in the JSON format that has a native support for booleans. But microdata is parsed from attributes/text content, so my question was how is boolean possible there?